dafny-lang/dafny
# Improve command-line dafny server

* Add command-line parameters `-encode`, `-plaintext`, and `-decode`. See `README.md` for a description.
* Change `selftest` command-line argument to `-selftest`
* Add “marshal” command that converts one plaintext program into a base64-encoded string
* Rename "counterExample" command "counterexample"
* Include temporarily excluded tests
* Check and update expected output
